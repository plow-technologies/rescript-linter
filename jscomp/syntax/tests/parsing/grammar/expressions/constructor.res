let x = Red

let y = Colors.Blue

// sugar for Rgb(())
let z = Rgb()

let v = Vertex(1., 2., 3., 4.)

let colour = Shades.Colors.White

let u = ()
