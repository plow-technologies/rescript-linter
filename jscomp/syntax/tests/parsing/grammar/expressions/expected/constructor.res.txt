let x = Red
let y = Colors.Blue
let z = Rgb ()
let v = Vertex (1., 2., 3., 4.)
let colour = Shades.Colors.White
let u = ()